ocrd/ocrd/processor/base.py

```diff
@@ -102,10 +102,11 @@ def generate_processor_help(ocrd_tool):
         parameter_help = '  NONE\n'
     else:
         for param_name, param in ocrd_tool['parameters'].items():
-            parameter_help += wrap_text('  "%s" [%s%s] %s' % (
+            parameter_help += wrap_text('  "%s" [%s%s%s] %s' % (
```

Comment:
or rather – since `required` is already mutually exclusive with `default`:

```suggestion
            parameter_help += wrap_text('  "%s" [%s%s] %s' % (
```